src/ios/CleverTapPlugin.m

```diff
@@ -972,7 +977,12 @@ - (void)getAllInboxMessages:(CDVInvokedUrlCommand *)command {
 - (void)getUnreadInboxMessages:(CDVInvokedUrlCommand *)command {
     
     [self.commandDelegate runInBackground:^{
-        NSArray *unreadInboxMessages = [clevertap getUnreadInboxMessages];
```

Comment:
@Sonal-Kachare Can we make a method that converts List of `CleverTapInboxMessage` Object to JSONArray to avoid a duplicate piece of code? Similar to - https://github.com/CleverTap/clevertap-flutter/blob/master/ios/Classes/CleverTapPlugin.m#L744